guide/english/certifications/javascript-algorithms-and-data-structures/basic-javascript/replace-loops-using-recursion/index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Replace Loops using Recursion
+---
+
+# Replace Loops using Recursion
+
+## Problem explanation
```

Comment:
We have been removing these as the instructions are already in the challenge itself.
```suggestion
```